zebra-chain/src/orchard/tree.rs

```diff
@@ -265,7 +252,7 @@ pub enum NoteCommitmentTreeError {
 }
 
 /// Orchard Incremental Note Commitment Tree
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug)]
 pub struct NoteCommitmentTree {
```

Comment:
Two options here:
- make a `SerializedNoteCommitmentTree` struct that has the same field order and similar types to the previous `bridgetree` crate, then derive `Serialize, Deserialize` on it ourselves
- manually implement the same format as bincode serialization

Here's the bincode format we need to match:
https://docs.rs/bincode/latest/bincode/index.html

Reply:
This comment is outdated now.